### PR TITLE
Fix/upgrade emotion/styled

### PR DESCRIPTION
**Pull request by ManjariRathore**
**What kind of change does this PR introduce?**'

This PR updates the version of the @emotion/styled package from 11.11.5 to 11.13.0, addresing issue #2137 

**Issue Number:**
#2137 

Fixes #

**Did you add tests for your changes?**

NA

**Snapshots/Videos:**

**If relevant, did you update the documentation?**
No

**Summary**

**Does this PR introduce a breaking change?**
No

**Other information**
NA

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
YES

## Summary by CodeRabbit

- **Documentation**
	- Improved clarity and organization of the `CONTRIBUTING.md` file.
	- Removed the "Conflict Resolution" section; integrated content into the "Branching Strategy."
	- Renamed the default branch from `main` to `master` and introduced a new "Alpha-x.x.x" branch for stability testing.
  
- **Dependency Updates**
	- Updated the `@emotion/styled` package version from `^11.11.5` to `^11.13.0`, which may enhance performance and introduce new features.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -63,19 +64,15 @@ Make sure you are following [issue report guidelines](ISSUE_GUIDELINES.md) avail
...
-When multiple developers are working on issues there is bound to be a conflict of interest (not to be confused with git conflicts) among issues, PRs or even ideas. Usually these conflicts are resolved in a **First Come First Serve** basis however there are certain exceptions to it.
+- Develop branch: For unstable code and bug fixing
+- Alpha-x.x.x: for stability teesting
+- Master: Where the stable production ready code lies
 
-- In the cases where you feel your potential issues could be an extension or in conflict with other PRs it is important to ask the author of the PR in the slack channel or in their PRs or issues themselves why he/she did not write code for something that would require minimal effort on their part.
-- Based on basic courtesy, it is good practice to let the person who created a function apply and test that function when needed.
-- Last but not the least, communication is important make sure to talk to other contributors, in these cases, in slack channel or in a issue/PR thread.
-- As a last resort the Admins would be responsible for deciding how to resolve this conflict.
+### Contributing Code
```
[the remark this answers is not in the document]
Comment (ManjariRathore) [reply CONTRIBUTING.md:71]: Updated in latest commit.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,8 +15,9 @@
     - [Issues](#issues)
     - [Pull Requests](#pull-requests)
     - [Branching Strategy](#branching-strategy)
-    - [Conflict Resolution](#conflict-resolution)
+    - [Branching Strategy](#branching-strategy-1)
   - [Contributing Code](#contributing-code)
+  - [Contributing Code](#contributing-code-1)
```
[the remark this answers is not in the document]
Comment (ManjariRathore) [reply CONTRIBUTING.md]: Updated in latest commit

File: CONTRIBUTING.md
Comment (palisadoes) [inline CONTRIBUTING.md:1]: Your PR has too many unnecessary files that don’t relate to your issue. You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD path/to/file
git commit

Use these commands to exclude this file from the PR

**Review by palisadoes — CHANGES_REQUESTED** [review]
Exclude CONTRIBUTING.md from the PR. It is out of scope.

Your PR has too many unnecessary files that don’t relate to your issue. You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD path/to/file
git commit

Use these commands to exclude this file from the PR

**Comment by ManjariRathore** [comment]
I apologize @palisadoes, I have raised a new PR.

**Pull request closed without merging**